### PR TITLE
Issue 19592 create component that preview material icons v3

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
preview icon selected added and component name rename

### CODE REVIEW

**Review by ivorpad — APPROVED** [review]

File: src/elements/dot-material-icon-picker/dot-material-icon-picker.tsx [hunk 1]
```diff
@@ -148,14 +151,20 @@ export class DotMaterialIcon {
...
             colorValue: this.colorValue,
             name: this.name,
-            value: this.value,
+            value: this.value
         });
     };
 
     render() {
         return (
             <div class="dot-material-icon">
```
Comment (fmontes) [inline src/elements/dot-material-icon-picker/dot-material-icon-picker.tsx:160]: If you need a wrapper you might want to use `<Host>`

**Review by fmontes — CHANGES_REQUESTED** [review]

File: src/elements/dot-material-icon-picker/dot-material-icon-picker.tsx [hunk 2]
```diff
@@ -148,14 +151,20 @@ export class DotMaterialIcon {
...
                 <div class="dot-material-icon__select-container">
+                    <mwc-icon
+                        class="dot-material-icon__preview"
+                        style={{ color: this.colorValue || '#000' }}
```
Comment (hmoreras) [inline src/elements/dot-material-icon-picker/dot-material-icon-picker.tsx:164]: is needed the `or` since `#000` is already the default value ?

**Review by hmoreras — CHANGES_REQUESTED** [review]

File: src/elements/dot-material-icon-picker/dot-material-icon-picker.scss [hunk 3]
```diff
@@ -1,16 +1,36 @@
...
 
 .dot-material-icon__select-container {
+    display: flex;
+    flex-direction: column;
     flex-grow: 1;
     margin-right: 16px;
     position: relative;
 }
 
+.dot-material-icon_select-input {
```
Comment (fmontes) [inline src/elements/dot-material-icon-picker/dot-material-icon-picker.scss:13]:
```suggestion
.dot-material-icon__select-input {
```

File: src/elements/dot-material-icon-picker/dot-material-icon-picker.scss [hunk 4]
```diff
@@ -19,15 +39,20 @@
...
+.dot-material-icon__option label {
+    display: flex;
+}
+
 .dot-material-icon__option mwc-icon {
-    --mdc-icon-size: 14px;
+    align-items: center;
+    display: flex;
+    --mdc-icon-size: 1em;
     margin-right: 8px;
```
Comment (fmontes) [inline src/elements/dot-material-icon-picker/dot-material-icon-picker.scss:56]: You can use `em` everywhere since the font-size is configurable.

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by ivorpad — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**